### PR TITLE
Add anyOf and oneOf support

### PR DESCRIPTION
**Pull request by Alan-Cha**

### CODE REVIEW

**Comment by laredo** [comment]
SIze of PR is due to Stripe API addition to the test suite

File: packages/openapi-to-graphql/package.json [hunk 1]
```diff
@@ -71,14 +71,15 @@
     "jsonpath-plus": "^0.18.0",
     "oas-validator": "^3.1.0",
     "request": "^2.88.0",
+    "rfdc": "^1.1.4",
```
Comment (Alan-Cha) [inline packages/openapi-to-graphql/package.json]: Is this being used?

File: packages/openapi-to-graphql/src/preprocessor.ts [hunk 2]
```diff
@@ -219,8 +223,7 @@ export function preprocessOas(
           oas
         }
 
-        // Handle operationId property name collision
-        // May occur if multiple OAS are provided
+        // Handle operationId property name collision // May occur if multiple OAS are provided
```
Comment (Alan-Cha) [inline packages/openapi-to-graphql/src/preprocessor.ts]: Fix comment.

**Comment by ErikWittern** [comment]
@Alan-Cha Thanks for making these additional improvements! This is a huge step for OtG!

**Pull request merged**